app/containers/collectionShow/reducer.tsx

```diff
@@ -1,33 +1,35 @@
 import { ACTION_TYPES, Actions } from '../../actions/actionTypes';
-import { AUTHOR_PAPER_LIST_SORT_TYPES } from '../common/sortBox';
+import { AUTHOR_PAPER_LIST_SORT_TYPES } from '../../components/common/sortBox';
+import { toggleElementFromArray } from '../../helpers/toggleElementFromArray';
+import { multiRemoveElementFromArray } from '../../helpers/multiRemoveElementFromArray';
 
 export interface CollectionShowState
   extends Readonly<{
       isLoadingCollection: boolean;
       pageErrorCode: number | null;
       isLoadingPaperToCollection: boolean;
-      isShareDropdownOpen: boolean;
       mainCollectionId: number;
       totalPaperListPage: number;
       currentPaperListPage: number;
       papersTotalCount: number;
       sortType: AUTHOR_PAPER_LIST_SORT_TYPES;
       searchKeyword: string;
       paperIds: number | number[];
+      selectedPaperIds: number[];
```

Comment:
you should initialize this property when the collection page is changed at `collectionShow component`.
I recommend using useEffect Hooks.